app/store/electronObjects.ts

```diff
@@ -1,4 +1,4 @@
 export const electronObjects = {
   views: {},
   window: null
-};
+};
```

Comment:
i also find it strange that terminating newlines are removed, i thought they were best practice still

Reply:
Prettier will do this automatically: https://prettier.io/docs/en/rationale.html#empty-lines